.github/workflows/ci.yml

```diff
@@ -62,7 +123,7 @@ jobs:
           poetry install --with dev --without docs --no-interaction -E slurm -E gunicorn $DB
 
       - name: Test with pytest
-        run: poetry run coverage run --concurrency=thread,greenlet,multiprocessing -m pytest
+        run: poetry run coverage run --concurrency=thread,greenlet,multiprocessing -m pytest tests/0[58]*
```

Comment:
I'd rather achieve the same split in a more explicit way, if possible.
See https://github.com/fractal-analytics-platform/fractal-tasks-core/blob/main/.github/workflows/ci_poetry.yml#L46C54-L46C74.

In the "core" block, we could use `pytest --ignore X --ignore Y`, and in the "job" block we could use `pytest folder1 folder2`